tests/py-flux-test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+from flux.thermal import PccThermalModel1D
+from flux.grids import setgrid
+
+class PyFluxTest(unittest.TestCase):
+    def test_conductionQ(self):
```

Comment:
rename to test_pcc_thermal_model_1d instead of test_conductionQ since we hide the name "conductionQ" from the user